upup/models/cloudup/_aws/topologies/_topology_privatemasters/network.yaml

```diff
@@ -0,0 +1 @@
+# TODO Kris - We need to built this out - according to Chris we have a legit use case for this :)
```

Comment:
Fixed in https://github.com/kubernetes/kops/pull/694/commits/319756ed9f71464804a62dce0869feff25f82086